source/common/network/dns_impl.cc

```diff
@@ -28,7 +28,16 @@ DnsResolverImpl::DnsResolverImpl(Event::Dispatcher& dispatcher)
   // launch via main().
   ares_library_init(ARES_LIB_INIT_ALL);
   ares_options options;
+
   initializeChannel(&options, 0);
+
+  if (resolvers.size() > 0) {
+    std::string resolvers_csv = StringUtil::join(resolvers, ",");
+    int result = ares_set_servers_ports_csv(channel_, resolvers_csv.c_str());
+    if (result != ARES_SUCCESS) {
+      throw EnvoyException(fmt::format("dns: error setting resolvers: {}", result));
```

Comment:
Ah yes, missed that part in the style guide. Fixed in https://github.com/lyft/envoy/pull/1018/commits/eb8cf596354e23ffe88b1bf508b0ae96dfe5edc8.